internal/cc/instance.go

```diff
@@ -0,0 +1,136 @@
+package cc
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+
+	"github.com/matrix-org/complement"
+	"github.com/matrix-org/complement-crypto/internal/api"
+	"github.com/matrix-org/complement-crypto/internal/config"
+	"github.com/matrix-org/complement-crypto/internal/deploy"
+	"github.com/matrix-org/complement/helpers"
+)
+
+// Instance represents a test instance (process
```

Comment:
Just a missing `)` or is there more to go here?

Reply:
Fixed